docs/examples/spectrum/average_photon_energy.py

```diff
@@ -0,0 +1,198 @@
+# import the required packages and define some basic system parameters
+# and meteorological conditions.
+
+# %%
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from scipy.integrate import trapezoid
+from pvlib import spectrum, solarposition, irradiance, atmosphere
+
+lat, lon = 39.742, -105.18  # NREL SRRL location
+tilt = 25
+azimuth = 180  # south-facing system
+pressure = 81190  # at 1828 metres AMSL, roughly
+water_vapor_content = 0.5  # cm
+tau500 = 0.1
+ozone = 0.31  # atm-cm
+albedo = 0.2
+
+times = pd.date_range('2023-01-01 08:00', freq='h', periods=9,
+                      tz='America/Denver')
+solpos = solarposition.get_solarposition(times, lat, lon)
+aoi = irradiance.aoi(tilt, azimuth, solpos.apparent_zenith, solpos.azimuth)
+
+relative_airmass = atmosphere.get_relative_airmass(solpos.apparent_zenith,
+                                                   model='kastenyoung1989')
+
+# %%
+# Spectral simulation
+# -------------------------
+# With all the necessary inputs now defined, we can model spectral irradiance
+# using :py:func:`pvlib.spectrum.spectrl2`. As we are calculating spectra for
+# more than one set of conditions, the function will return a dictionary
+# containing 2-D arrays for the spectral irradiance components and a 1-D array
+# of shape (122,) for wavelength. For each of the 2-D arrays, one dimension is
+# for wavelength in nm and one is for irradiance in Wm⁻²nm⁻¹.
+
+spectra_components = spectrum.spectrl2(
+    apparent_zenith=solpos.apparent_zenith,
+    aoi=aoi,
+    surface_tilt=tilt,
+    ground_albedo=albedo,
+    surface_pressure=pressure,
+    relative_airmass=relative_airmass,
+    precipitable_water=water_vapor_content,
+    ozone=ozone,
+    aerosol_turbidity_500nm=tau500,
+)
+
+# %%
+# Visualising the spectral data
+# -----------------------------
+# Let's take a look at the spectral irradiance data simulated hourly for eight
+# hours on the first day of 2023.
+
+plt.figure()
+plt.plot(spectra_components['wavelength'], spectra_components['poa_global'])
+plt.xlim(200, 2700)
+plt.ylim(0, 1.8)
+plt.ylabel(r"Spectral irradiance (Wm⁻²nm⁻¹)")
+plt.xlabel(r"Wavelength (nm)")
+time_labels = times.strftime("%H:%M")
+labels = [
+    "{}, AM {:0.02f}".format(*vals)
+    for vals in zip(time_labels, relative_airmass)
+]
+plt.legend(labels)
+plt.show()
+
+# %%
+# Given the changing broadband irradiance throughout the day, it is not obvious
+# from inspection how the relative distribution of light changes as a function
+# of wavelength. We can normalise the spectral irradiance curves to visualise
+# this shift in the shape of the spectrum over the course of the day. In
+# this example, we normalise by dividing each spectral irradiance value by the
+# total broadband irradiance, which we calculate by integrating the entire
+# spectral irradiance distribution with respect to wavelength.
+
+poa_global = spectra_components['poa_global']
```

Comment:
I suggest naming this variable `spectral_poa` or something similar.  `poa_global` by itself could easily be confused for a broadband value.